users/get-user-info-by-X.sh

```diff
@@ -0,0 +1,15 @@
+# Get sorted list of User Roles - Search by email
+gc users list --expand="authorization" | jq -cr 'first(.[] | select(.email == "user.email@example.com")) | .authorization.roles[].name' | sort --ignore-case
+
+########
```

Comment:
You could argue that these are general composition strats.  And really just cli foo.  Can either promote this to a generic section, leave it here, or remove it entirely.

Also assumes that y'all like this pattern to reduce data volume at the expense of multiple calls.

Reply:
Dude. I love it.  I had never even thought about using the first() call to avoid having to search the entire stack.